src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -1012,6 +1012,20 @@ public static void onEmptyLeftClick(EntityPlayer player, ItemStack stack)
         MinecraftForge.EVENT_BUS.post(new PlayerInteractEvent.LeftClickEmpty(player, stack));
     }
 
+    public static net.minecraftforge.event.entity.player.CriticalHitEvent getCriticalHit(EntityPlayer player, Entity target, boolean vanillaCritical)
+    {
+        net.minecraftforge.event.entity.player.CriticalHitEvent hitResult = new net.minecraftforge.event.entity.player.CriticalHitEvent(player, target);
+        net.minecraftforge.common.MinecraftForge.EVENT_BUS.post(hitResult);
+        if(hitResult.getResult() != Event.Result.DENY)
+        {
+            if(hitResult.getResult() == Event.Result.ALLOW || vanillaCritical)
+            {
+                return hitResult;
+            }
+        }
```

Comment:
But it was requested to make even normal hits to critical ones. So I am using the Result stuff.